pkg/multiTenancyPlugins/naming/nameScoping.go

```diff
@@ -59,6 +60,9 @@ func (nameScoping *DefaultNameScopingImpl) Handle(command utils.CommandEnum, clu
 			newQuery = strings.Replace(r.RequestURI, r.URL.Query().Get("name"), r.URL.Query().Get("name")+r.Header.Get(headers.AuthZTenantIdHeaderName), 1)
 			config.Config.Labels[headers.OriginalNameLabel] = r.URL.Query().Get("name")
 		}
+		// Replace network reference with network ID, force Network-scoped alias for DNS use.
```

Comment:
But this is not the case here - you get teh --name parameter  into the config. Thi swill be non empty only in the if , is it ?
Also we do support somone doing --net-alias as this goes directly to the engine